src/Sarif.Sdk.sln

```diff
@@ -24,6 +24,48 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.FunctionalTests", "Sa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.Multitool.FunctionalTests", "Sarif.Multitool.FunctionalTests\Sarif.Multitool.FunctionalTests.csproj", "{859920F3-DB09-4C1D-9EAD-DFB1D7A03A46}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "root", "root", "{BBDE0751-B8A3-4EA6-A3DF-3CE7BA41EA0F}"
+	ProjectSection(SolutionItems) = preProject
+		..\.gitattributes = ..\.gitattributes
```

Comment:
>gitattributes [](start = 6, length = 13)

Add solution items for all loose files. All of them are grouped under a single top-level solution folder `root` and its subfolders `src` and `scripts`, so as not to pollute Solution Explorer's view.